config/o11y/o11y.go

```diff
@@ -36,18 +36,8 @@ type Config struct {
 	StatsdTelemetryDisabled bool
 }
 
-// Setup is the primary entrypoint to initialise the o11y system both in development and production.
-//
-// To support projects with monobinaries, DevInit can be called first.
+// Setup is the primary entrypoint to initialise the o11y system.
 func Setup(ctx context.Context, o Config) (context.Context, func(context.Context), error) {
-	if coordinator == nil {
```

Comment:
I think it is fair to push this onto the clients of this package if they have a combined dev binary